### PR TITLE
Add'l params for OSP. BZ 1758114.

### PR DESCRIPTION
**Pull request by maxwelldb**
Adding details for OSP. See https://bugzilla.redhat.com/show_bug.cgi?id=1758114

@morenod Hi! Does this resolve what you found in the BZ ticket?

### CODE REVIEW

File: modules/installation-configuration-parameters.adoc [hunk 1]
```diff
@@ -63,12 +63,12 @@ uses the `<metadata.name>.<baseDomain>` format.
 |`controlPlane.platform`
 |The cloud provider to host the control plane machines. This parameter value
 must match the `compute.platform` parameter value.
-|`aws`, `azure`, or `gcp`
+|`aws`, `azure`, `gcp`, or, for OpenStack, `{}`
```
Comment (kalexand-rh) [inline modules/installation-configuration-parameters.adoc]: This change invalidates the descriptions for these parameters, at least for OpenStack.

Comment (maxwelldb) [reply modules/installation-configuration-parameters.adoc]: Fair point. 

@morenod Does the OSP value here definitely not match as the others do?

Comment (morenod) [reply modules/installation-configuration-parameters.adoc]: when compute.platform is empty (nothing to say about it), value is {}.
when it has content, value is the name of the platform.

this is valid for aws and openstack (I cannot confirm the rest of the platforms as I havent tested them)

Comment (maxwelldb) [reply modules/installation-configuration-parameters.adoc]: @kalexand-rh So, it seems like controlPlane.platform and compute.platform can also have {} as a platform-independent value. Can you confirm this?

Comment (kalexand-rh) [reply modules/installation-configuration-parameters.adoc]: > when compute.platform is empty (nothing to say about it), value is {}.
> when it has content, value is the name of the platform.

@abhinavdahiya, can you comment on the intended usage of `{}` as a `compute.platform` value in the `install-config.yaml` file? I thought it was intended to blank out the other platform variables and that the `compute.platform` and `controlPlane.platform` parameter values needed to match.

Comment (abhinavdahiya) [reply modules/installation-configuration-parameters.adoc]: > https://github.com/openshift/installer/blob/master/docs/user/openstack/customization.md#machine-pools

it seems like openstack does have compute level properties.

Comment (abhinavdahiya) [reply modules/installation-configuration-parameters.adoc]: ex: https://github.com/openshift/installer/blob/master/docs/user/openstack/customization.md#custom-machine-pools

Comment (maxwelldb) [reply modules/installation-configuration-parameters.adoc]: @abhinavdahiya So is it the case that these two fields don't necessarily have to match like the docs currently state?

**Comment by maxwelldb** [comment]
@morenod Does this PR now resolve your BZ ticket?

**Comment by morenod** [comment]
@maxwelldb lgtm

**Comment by maxwelldb** [comment]
Thanks!

**Comment by maxwelldb** [comment]
@bergerhoffer @lamek @bobfuru Can I get an :+1: / :-1: on this whenever you have a moment? David's both the reporter and QE contact, so this should be good up to that point.

**Review by bergerhoffer — COMMENTED** [review]
LGTM

**Pull request merged**